### ISSUE TITLE
Unable to get the state of a messae

### ISSUE THREAD

**Issue by AhyaanTech** [minghuaw/azservicebus#25]
```
registry\src\index.crates.io-6f17d22bba15001f\azservicebus-0.20.1\src\primitives\service_bus_peeked_message.rs:344:22:
internal error: entered unreachable code: Expecting a Long
```

I dont know what additional details that i can give, but i can follow instructions to give you more information.

The message was created using web azure service bus explorer from the azure portal. and when i peek that same message from the web i do see the active state as well

**Comment by minghuaw** [comment]
This was probably my oversight. Should've covered all numeric types

**Comment by minghuaw** [comment]
Should be fixed in #26. Feel free to re-open the issue if it is not solved

**Comment by AhyaanTech** [comment]
Its Fixed! thank you and much appreciated!
